pyschism/forcing/hycom/hycom2schism.py

```diff
@@ -177,7 +181,7 @@ def __init__(self, hgrid, vgrid=None):
         self.hgrid = hgrid
         self.vgrid = Vgrid.default() if vgrid is None else vgrid
 
-    def fetch_data(self, outdir: Union[str, os.PathLike], start_date, rnday, ocean_bnd_ids = [0], elev2D=True, TS=True, UV=True, restart=False, adjust2D=False, lats=None, msl_shifts=None): 
+    def fetch_data(self, outdir: Union[str, os.PathLike], start_date, rnday, ocean_bnd_ids = [0], elev2D=True, TS=True, UV=True, restart=False, adjust2D=False, lats=None, msl_shifts=None):
```

Comment:
You don't need 2 flags for the same thing (msl_shifts and adjust2D). Use only one which gets applied if it's not `None`.